target/linux/ath79/dts/qca9533_tplink_cpexxx.dtsi

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+		label-mac-device = &eth0;
```

Comment:
You use eth0 here, but only have wmac defined in the parent DTSI. Since eth0 and wmac are the same, it would be easiest to just use wmac for label-mac-device here (and add a short comment about the change in commit message).

Reply:
This compiles cleanly and works when tested on my CPE210 V2 and CPE220 V3

Reply:
Can you post the result of `ls /proc/device-tree/leds/` on one of the devices?
Does the led4 blink during boot?

The above is just to satisfy my curiosity. To my knowledge, OpenWrt consequently uses the node referencing scheme as I pointed out in my comments. So even if your version is working, I'd insist on using the &leds variant.

Reply:
Yes led4 blinks during boot here is the output of the ls -al
drwxr-xr-x    8 root     root             0 Jan  1  1970 .
drwxr-xr-x   10 root     root             0 Jan  1  1970 ..
-r--r--r--    1 root     root            10 Oct 26 21:06 compatible
drwxr-xr-x    2 root     root             0 Oct 26 21:06 lan0
drwxr-xr-x    2 root     root             0 Oct 26 21:06 lan1
drwxr-xr-x    2 root     root             0 Oct 26 21:06 link1
drwxr-xr-x    2 root     root             0 Oct 26 21:06 link2
drwxr-xr-x    2 root     root             0 Oct 26 21:06 link3
drwxr-xr-x    2 root     root             0 Jan  1  1970 link4
-r--r--r--    1 root     root             5 Oct 26 21:06 name